### DISCUSSION TITLE
军需商店兑换里能否添加金紫布里的选项？

### DISCUSSION THREAD

**Discussion by Jacobu2077** [LmeSzinc/AzurLaneAutoScript#3379] [General]
我看到商店过滤词里只有bulin，没有区分金色和紫色，本身自己不缺紫布里有点缺金布里，但是每次功勋商店里都会自动购买紫布里导致功勋不够买金布里

**Comment by guoh064** [comment]
https://github.com/LmeSzinc/AzurLaneAutoScript/wiki/reward_shop_filter_string#tiers

**Reply by Jacobu2077** [reply]
万分感谢！
